src/components/ViewDeck/ViewDeck.js

```diff
@@ -97,228 +80,205 @@ class ViewDeck extends Component {
-      if (card.type_line.includes("land")) {
+      if (card.jsonData.type_line.includes("land")) {
         //console.log('this is a land and shouldnt be counted');
       } else {
-        convertedManaCosts.push(card.cmc);
-        //console.log(convertedManaCosts);
+        convertedManaCosts.push(card.jsonData.cmc);
       }
     });
 
+    //NOT SURE WHAT THIS WAS MEANT TO DO ORIGIONALLY
     convertedManaCosts.forEach((cost) => {
       //console.log(cost);
     });
     return convertedManaCosts;
   };
 
-  render() {
-    const includedCards = this.props.reduxStore.cardList.filter(
-      (card) => card.deckid === this.props.reduxStore.selectedDeck.id
-    );
-    console.log("includedCards:", includedCards);
-    const featuredCard = includedCards.filter(
-      (featured) => featured.is_featured === true
-    );
-    const parsedFeaturedCard = featuredCard.map((fCard) =>
-      JSON.parse(fCard.api_data)
-    );
-    console.log("parsedFeaturedCard", parsedFeaturedCard);
-    const featuredUri = parsedFeaturedCard.map(
-      (fUri) => fUri.image_uris.normal
-    );
-
-    this.countQty(includedCards);
-
-    const parsedCards = this.parseCards(includedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 168 ~ ViewDeck ~ render ~ parsedCards",
-      parsedCards
-    );
-
-    const devotion = this.devotion(parsedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 204 ~ ViewDeck ~ render ~ devotion",
-      devotion
-    );
-
-    const cmc = this.cmc(parsedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 215 ~ ViewDeck ~ render ~ cmc sans land",
-      cmc
-    );
-
-    return (
-      <div>
-        <h1 className="editDeckHeader">
-          Viewing {this.props.reduxStore.selectedDeck.deckname} from{" "}
-          {this.props.reduxStore.user.username}{" "}
-        </h1>
-        {/* <h1 id="editDeckHeader">Total Cards: {this.state.deckQty}</h1> */}
-
-        <div className="viewDeckEditDeckView">
-          <div>
-            <img
-              className="featuredViewDeckCard"
-              src={this.state.hoverCard}
-              width="200px"
-              height="280"
-            />
-          </div>
-
-          <br />
-          <h4 className="editDeckHeader">Total Cards: {this.state.deckQty}</h4>
+  const includedCards = props.reduxStore.cardList.filter(
+    (card) => card.deckid === props.reduxStore.selectedDeck.id
+  );
+  // console.log("includedCards:", includedCards);
+  const featuredCard = includedCards.filter(
+    (featured) => featured.is_featured === true
+  );
+  const parsedFeaturedCard = featuredCard.map((fCard) =>
+    JSON.parse(fCard.api_data)
+  );
+  // console.log("parsedFeaturedCard", parsedFeaturedCard);
+  const featuredUri = parsedFeaturedCard.map((fUri) => fUri.image_uris.normal);
+
+  countQty(includedCards);
+
+  const parsedCards = parseCards(includedCards);
+
+  const devotion = devotionCalc(parsedCards);
+
+  const cmc = cmcCalc(parsedCards);
```

Comment:
This is again not good. You are setting variables with written methods on every single render.
https://reactjs.org/docs/hooks-effect.html